ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -92,7 +92,7 @@ export default class AndroidGenerator implements ContainerGenerator {
     let mustacheView: any = {
       customPermissions: [],
       customRepos: [],
-      permissions: []
+      permissions: [],
```

Comment:
Yay for trailing comma 👍 Can we make this a default?

Reply:
Already the case, enforced by prettier on pre-commit hook. Got bypassed by the external contributor that changed this file (https://github.com/electrode-io/electrode-native/pull/1372).. didn't pay too much attention on this one (didn't even squash merged).